guides/upstream_filename_to_satellite_link.json

```
@@ -2,6 +2,7 @@
   "build/Administering_Project/index-satellite.html": "administering_red_hat_satellite",
   "build/Configuring_Load_Balancer/index-satellite.html": "configuring_capsules_with_a_load_balancer",
   "build/Configuring_User_Authentication/index-satellite.html": "configuring_authentication_for_red_hat_satellite_users",
+  "build/Configuring_Virtual_Machine_Subscriptions/index-satellite.html": "configuring_virt-who_for_virtual_machine_subscriptions",
```

```suggestion
  "build/Configuring_Virt-Who-VM-Subscriptions/index-satellite.html": "configuring_virt-who_for_virtual_machine_subscriptions",
```
This path has to match the folder name after the `doc-` prefix.